charts/stable/pod-gateway/README_CHANGELOG.md.gotmpl

```diff
@@ -13,7 +13,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Added
 
-- Initial version
+- renamed chart
```

Comment:
This seems "weird" for a new chart... Maybe mention here as well that this is the successor to `pod-gateway-setter`?

Reply:
done, thanks for the review.